test/Microsoft.AspNet.Mvc.Core.Test/VirtualFileResultTest.cs

```diff
@@ -262,21 +262,23 @@ public void Constructor_SetsContentTypeAndParameters()
         {
             // Arrange
             // Point the IFileProvider root to a different subfolder
-            var fileProvider = new PhysicalFileProvider(Path.GetFullPath("./Properties"));
-            var filePathResult = new VirtualFileResult(path, "text/plain")
+            using (var fileProvider = new PhysicalFileProvider(Path.GetFullPath("./Properties")))
```

Comment:
@pranavkm I'm not too worried about this somewhat-unrelated suggestion. Are you?